test/unit/tile.js

```diff
@@ -297,6 +319,7 @@ describe('Tile', function () {
           assert.strictEqual(2225, info.height);
           assert.strictEqual(3, info.channels);
           assert.strictEqual('undefined', typeof info.size);
+          assertTileOverlap(directory, 512);
```

Comment:
The assertions within `assertTileOverlap()` are async so that function should probably accept a callback parameter to  allow the call to `assertDeepZoomTiles()` be be deferred until after it has completed.

Reply:
Makes total sense. To remedy this, I've added the callback parameter and placed `assertTileOverlap()` in the callback for `assertDeepZoomTiles()`. I thought it made the most sense to check the tile overlap after `assertDeepZoomTiles()`.